main.go

```diff
@@ -71,7 +71,7 @@ func (e *Exporter) scrape() (up float64, prj *compute.Project, rgl *compute.Regi
 	regionList, err := e.service.Regions.List(e.project).Do()
 	if err != nil {
 		log.Errorf("Failure when querying region quotas: %v", err)
-		return 0, nil, nil
+		return 0, project, nil
```

Comment:
@Duologic one thing that came up to me after initial approval.

While i like the idea of returning the `project` when that part succeded ... i am not sure we should return `up` as 0 here or as `1` ?

On one side the Regions request failed so it would make sense to return 0 , on the other side though we are returning some data , for the project, so returning up as `0` might be misleading ?

Reply:
Ok I will approve and merge as it is.

I think is fair to consider the scrape as unealthy if any of the scrapes fails

Reply:
There is an argument in there that bit-wise doesn't cover the load, a more granular `up`, `partial` and `down` approach would make more sense, but I don't see much benefit in the extra effort.